tox.ini

```diff
@@ -323,7 +325,7 @@ deps_files = dev_requirements.txt
 docker =
 commands =
     flake8
-    black --target-version py36 --check --diff .
+    black --target-version py37 --check --diff .
```

Comment:
black --target-version py36 --check --diff .

Reply:
I've dropped support for Python 3.6 since it's EOL and aioredis should only support 3.7+ (due to event loop handling).

Reply:
@Andrew-Chen-Wang For here, let's keep python 3.6 in here (for now). There's a lot of downstream impact that I'm happy to do - but will be future. Sound reasonable? I changed the black check accordingly.